databricks/koalas/series.py

```diff
@@ -4975,6 +4975,49 @@ def item(self):
         """
         return self.head(2).to_pandas().item()
 
+    def iteritems(self):
+        """
+        Lazily iterate over (index, value) tuples.
+
+        This method returns an iterable tuple (index, value). This is
+        convenient if you want to create a lazy iterator.
+
+        .. note:: Unlike pandas', the iteritems in Koalas returns only the number of items
+            specified in `ks.options.compute.max_rows`. If you want to get all items of
+            Series, specify the `ks.options.compute.max_rows` to the length of Series,
+            or just use pandas' directly with `to_pandas()`.
+
+        Returns
+        -------
+        iterable
+            Iterable of tuples containing the (index, value) pairs from a
+            Series.
+
+        See Also
+        --------
+        DataFrame.items : Iterate over (column name, Series) pairs.
+        DataFrame.iterrows : Iterate over DataFrame rows as (index, Series) pairs.
+
+        Examples
+        --------
+        >>> kser = ks.Series(range(10000))
+        >>> with ks.option_context("compute.max_rows", 5):
+        ...     items = kser.iteritems()
+        >>> for index, value in items:
+        ...     print("Index : {}, Value : {}".format(index, value))
+        Index : 0, Value : 0
+        Index : 1, Value : 1
+        Index : 2, Value : 2
+        Index : 3, Value : 3
+        Index : 4, Value : 4
+        """
+        kser = self[: ks.options.compute.max_rows]
+        return zip(iter(kser.index.to_numpy()), kser.values)
```

Comment:
Can you follow the implementation of `DataFrame.iterrows`?

Reply:
@HyukjinKwon 

Oops, sorry for the check late.

I'll manage this. Thanks!!